### PR TITLE
Portuguese translation for help-fastapi.md

### PR DESCRIPTION
**Pull request by pingu1m**

### CODE REVIEW

File: docs/pt/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Help FastAPI - Get Help
```
Comment (cassiobotaro) [inline docs/pt/docs/help-fastapi.md:1]:
```suggestion
# Ajuda FastAPI - Obtendo Ajuda
```

File: docs/pt/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Gostaria de ajudar o FastAPI, outros usuários, e o autor?
+
+Ou gostaria de obter ajuda para utilizar o **FastAPI**?
+
+Há maneiras muito simples de ajudar (algumas envolvem somente um ou dois cliques). 
+
+Há várias maneiras de buscar ajuda também.
+
+## Star **FastAPI** no Github
```
Comment (cassiobotaro) [inline docs/pt/docs/help-fastapi.md:13]:
```suggestion
## Estrele **FastAPI** no Github
```

File: docs/pt/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Você pode "star" FastAPI no GitHub (clicando no botão com a estrela no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
+
+Ao adicionar uma estrela, outros usuários serão capazes de encontrá-lo mais facilmente e ver que o framework foi útil à outros usuários.
+
+## Watch por releases o repository no GitHub
```
Comment (cassiobotaro) [inline docs/pt/docs/help-fastapi.md:19]:
```suggestion
## Aguarde por releases no repositório do GitHub
```

File: docs/pt/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+## Vote pelo FastAPI
+
+* <a href="https://github.com/vinta/awesome-python/pull/1209" class="external-link" target="_blank">Vote pare incluir o **FastAPI** in `awesome-python`</a>.
+* <a href="https://www.slant.co/options/34241/~fastapi-review" class="external-link" target="_blank">Vote pelo **FastAPI** no Slant</a>.
+
+## Ajude outros com issues no GitHub
+
+Você pode ver <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">as issues existentes</a> e tentar ajudar os outros.
+
+## Watch o repository no GitHub
```
Comment (cassiobotaro) [inline docs/pt/docs/help-fastapi.md:79]:
```suggestion
## Observe o repositório no GitHub
```

File: docs/pt/docs/help-fastapi.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+* <a href="https://github.com/tiangolo" class="external-link" target="_blank">Me seguir no **GitHub**</a>.
+    * Ver outros projetos Open Source que eu criei que pode lhe ajudar.
+    * Me seguir e ver quando eu criar um novo projeto Open Source.
+* <a href="https://twitter.com/tiangolo" class="external-link" target="_blank">Me seguir no **Twitter**</a>.
+    * Me falar como você usa o FastAPI (Eu gostaria muito de saber).
+    * Fazer perguntas.
+* <a href="https://www.linkedin.com/in/tiangolo/" class="external-link" target="_blank">conecte-se comigo no **Linkedin**</a>.
+    * Falar comigo.
+    * Me recomende ou Endosse :)
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:51]:
```suggestion
    * Me recomende ou me endosse :)
```

File: docs/pt/docs/help-fastapi.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+* <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">Ler oque escrevo (ou me seguir) no **Medium**</a>.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:52]:
```suggestion
* <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">Ler o que escrevo (ou me seguir) no **Medium**</a>.
```

File: docs/pt/docs/help-fastapi.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    * Ler outras ideias, artigos e ferramentas que eu criei.
+    * Me seguir para ver quando eu publico algo novo.
+
+## Tweet sobre **FastAPI**
+
+<a href="https://twitter.com/compose/tweet?text=Estou amando o FastAPI porque... https://github.com/tiangolo/fastapi cc @tiangolo" class="external-link" target="_blank">Tweet sobre **FastAPI**</a> e deixe eu e os outros saberem porque você gosta do **FastAPI**.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:58]:
```suggestion
<a href="https://twitter.com/compose/tweet?text=Estou amando o FastAPI porque... https://github.com/tiangolo/fastapi cc @tiangolo" class="external-link" target="_blank">Tweet sobre **FastAPI**</a> e deixe eu e outros saberem porque você gosta do **FastAPI**.
```

File: docs/pt/docs/help-fastapi.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+## Me fale como esta usando o **FastAPI**
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:60]:
```suggestion
## Me fale como está usando o **FastAPI**
```

File: docs/pt/docs/help-fastapi.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+Eu amaria saber como o **FastAPI** esta sendo usado, oque você gostou nele, e em qual projeto/empresa esta sendo usado, etc.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:62]:
```suggestion
Eu adoraria saber como o **FastAPI** está sendo usado, o que você gostou dele, e em qual projeto/empresa está sendo utilizado etc.
```

File: docs/pt/docs/help-fastapi.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+Você pode me falar:
+
+* <a href="https://twitter.com/compose/tweet?text=Oi @tiangolo, Estou usando o FastAPI na..." class="external-link" target="_blank">No **Twitter**</a>.
+* <a href="https://www.linkedin.com/in/tiangolo/" class="external-link" target="_blank">No **Linkedin**</a>.
+* <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">No **Medium**</a>.
+
+## Vote pelo FastAPI
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:70]:
```suggestion
## Vote no FastAPI
```

File: docs/pt/docs/help-fastapi.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+* <a href="https://github.com/vinta/awesome-python/pull/1209" class="external-link" target="_blank">Vote pare incluir o **FastAPI** in `awesome-python`</a>.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:72]:
```suggestion
* <a href="https://github.com/vinta/awesome-python/pull/1209" class="external-link" target="_blank">Vote pare incluir o **FastAPI** no _`awesome-python`_</a>.
```

File: docs/pt/docs/help-fastapi.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+* <a href="https://www.slant.co/options/34241/~fastapi-review" class="external-link" target="_blank">Vote pelo **FastAPI** no Slant</a>.
+
+## Ajude outros com issues no GitHub
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:75]:
```suggestion
## Ajude outros com as _issues_ no GitHub
```

File: docs/pt/docs/help-fastapi.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+Você pode ver <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">as issues existentes</a> e tentar ajudar os outros.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:77]:
```suggestion
Você pode ver <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">as _issues_ existentes</a> e tentar ajudar os outros.
```

File: docs/pt/docs/help-fastapi.md (hunk 4 above)
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:79]:
```suggestion
## Fique de olho no repositório no GitHub
```

File: docs/pt/docs/help-fastapi.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+## Watch o repository no GitHub
+
+Você pode "watch" o FastAPI no GitHub (clicando no botão escrito 'Watch' no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:81]:
```suggestion
Você pode "ficar de olho" no FastAPI no GitHub (clicando no botão escrito 'Watch' no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
```

File: docs/pt/docs/help-fastapi.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+Se você selecionar "Watching" ao invés de "Releases only" você ira receber notificações quando alguém criar uma issue.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:83]:
```suggestion
Se você selecionar "Watching" ao invés de "Releases only" você receberá notificações quando alguém criar uma _issue_.
```

File: docs/pt/docs/help-fastapi.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+Ai você pode tentar ajudar ele/ela a resolver suas issues.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:85]:
```suggestion
Ai você pode tentar ajudar ele/ela a resolver suas _issues_.
```

File: docs/pt/docs/help-fastapi.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+
+## Criar issues
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:87]:
```suggestion
## Criar _issues_
```

File: docs/pt/docs/help-fastapi.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+Você pode <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="external-link" target="_blank">criar uma nova issue</a> no repositório do GitHub, por exemplo para:
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:89]:
```suggestion
Você pode <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="external-link" target="_blank">criar uma nova _issue_</a> no repositório do GitHub, por exemplo para:
```

File: docs/pt/docs/help-fastapi.md [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+
+* Reportar uma bug/issue.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:91]:
```suggestion
* Reportar um _bug/issue_.
```

File: docs/pt/docs/help-fastapi.md [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+* Sugerir um novo recurso.
+* Fazer uma pergunta.
+
+## Criar um Pull Request
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:95]:
```suggestion
## Criar um _Pull Request_
```

File: docs/pt/docs/help-fastapi.md [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+
+Você pode <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">criar um Pull Request</a>, por exemplo para:
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:97]:
```suggestion
Você pode <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">criar um _Pull Request_</a> para, por exemplo:
```

File: docs/pt/docs/help-fastapi.md [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+
+* Corrigir um erro na documentação.
+* Sugerir novas seções na documentação.
+* Corrigir uma issue/bug existente.
```
Comment (Serrones) [inline docs/pt/docs/help-fastapi.md:101]:
```suggestion
* Corrigir um _bug/issue_ existente.
```

**Review by Serrones — CHANGES_REQUESTED** [review]

File: docs/pt/docs/help-fastapi.md (hunk 2 above)
Comment (pingu1m) [reply docs/pt/docs/help-fastapi.md:13]: Acredito que Favorite se encaixa melhor no contexto. @cassiobotaro o que vc acha?

File: docs/pt/docs/help-fastapi.md (hunk 22 above)
Comment (pingu1m) [reply docs/pt/docs/help-fastapi.md:101]: Por que dos _ ?

File: docs/pt/docs/help-fastapi.md (hunk 2 above)
Comment (pingu1m) [reply docs/pt/docs/help-fastapi.md:13]: Pessoal acredito que Star e Watch sejam termos como proprios do GitHub. Similares al Pull Request. 
Por exemplo aqui na documentacao em pt https://help.github.com/pt/github/getting-started-with-github/be-social
Note que o termo Watch esta sendo traduzido como Watch em negrito. Acredito, que o mesmo se aplique a o termo Star e Fork

Comment (cassiobotaro) [reply docs/pt/docs/help-fastapi.md:13]: Sim, concordo

File: docs/pt/docs/help-fastapi.md [hunk 23]
```diff
@@ -0,0 +1,106 @@
...
+Você gosta do **FastAPI**?
+
+Gostaria de ajudar o FastAPI, outros usuários, e o autor?
+
+Ou gostaria de obter ajuda para utilizar o **FastAPI**?
+
+Há maneiras muito simples de ajudar (algumas envolvem somente um ou dois cliques). 
+
+Há várias maneiras de buscar ajuda também.
+
```
Comment (rsip22) [inline docs/pt/docs/help-fastapi.md:12]: Is there a particular reason why the fastapi Twitter handle (and the suggestion to follow) has been removed from here? Even though the tweets are probably in English (I'm guessing), Twitter itself has a built-in translator for the tweets... so even people who only speak Portuguese may end up being interested in following it.

File: docs/pt/docs/help-fastapi.md [hunk 24]
```diff
@@ -0,0 +1,106 @@
...
+## Fique de olho nos lançamentos de novos recursos
+
+Você pode "_watch_" o FastAPI no GitHub (clicando no botão escrito 'Watch' no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
+
+Depois de clicar o botão é só selecionar _"Releases only"_
+
+Fazendo isso, você vai receber notificações (no seu e-mail) todas as vezes que tiver um novo lançamento (uma nova versão) do **FastAPI** com correções de _bugs_ e novos recursos.
+
+## Entre no _chat_
```
Comment (rsip22) [inline docs/pt/docs/help-fastapi.md:27]: This section has been removed on the original documentation. The Gitter chat is no longer recommended...

File: docs/pt/docs/help-fastapi.md [hunk 25]
```diff
@@ -0,0 +1,106 @@
...
+
+## Criar um _Pull Request_
+
+Você pode <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">criar um _Pull Request_</a> para, por exemplo:
+
+* Corrigir um erro na documentação.
+* Sugerir novas seções na documentação.
+* Corrigir um _bug/issue_ existente.
+* Adicionar um novo recurso.
+
```
Comment (rsip22) [inline docs/pt/docs/help-fastapi.md:103]: There are some sections of the original documentation missing here, I'm guessing they were added later?

- Join the chat (which now recommends Discord)
- Sponsor the author
- Sponsor the tools that power FastAPI

Would you consider adding those as well?

Other than that, excelent work!

**Comment by LorhanSohaky** [comment]
Duplicated #4583

**Comment by LorhanSohaky** [comment]
@tiangolo , can you close this PR?

**Comment by tiangolo** [comment]
Thanks @LorhanSohaky! I'll close this one. 🤓

**Pull request closed without merging**